src/he/ssl_ctx.c

```diff
@@ -289,10 +289,11 @@ he_return_code_t he_ssl_ctx_start_server(he_ssl_ctx_t *ctx) {
   }
 
 #ifndef HE_NO_PQC
-  int SERVER_CURVE_PQC_GROUPS[4] = {WOLFSSL_P521_KYBER_LEVEL5, WOLFSSL_P256_KYBER_LEVEL1,
-                                    WOLFSSL_ECC_SECP256R1, WOLFSSL_ECC_X25519};
+  int SERVER_CURVE_PQC_GROUPS[6] = {WOLFSSL_P521_ML_KEM_1024, WOLFSSL_P521_KYBER_LEVEL5,
```

Comment:
Do we need mlkem 512?

Reply:
Removed it, just realised in no way we will be using it lol